openshift_scalability/config/golang/podVertical.yaml

```diff
@@ -0,0 +1,21 @@
+provider: local
+ClusterLoader:
+  cleanup: true
+  projects:
+    - num: 1
+      basename: clusterproject
+      tuning: default
+      ifexists: delete
+      pods:
+        - num: 3000
+          image: gcr.io/google_containers/pause-amd64:3.0
+          basename: pausepods
+          file: pod-pause.json
+  tuningsets:
+    - name: default
+      pods:
+        stepping:
+          stepsize: 150
```

Comment:
The stepsize is increased 3 times, compared to 50 in node vertical, I am assuming it is to keep overall test duration short.  Have we tested if we can repeatably sustain deploying 150 pods with 60 sec pause time, I am assuming this will be spread across multiple compute nodes.

Reply:
great question.  and also from a procedural standpoint it seems like we merge tests before they are themselves tests.  I don't have a huge objection to this (or suggested fix), but it does raise valid questions like @wabouhamad raised.  For now, if it does cause problems, we follow up with another PR tweaking it to the values that ended up working?

Reply:
We have tripled the step size to keep the test suration short. I have tested this on a 5 node aws cluster and the cluster loader was okay with creating 150 pods at a time and it went on till we ran out of resources on the cluster to create more pods.